core/pom.xml

```diff
@@ -542,12 +542,18 @@ limitations under the License.
            javadoc on private methods. The configuration for "mvn site" is
            under "reporting", and is more lenient. -->
       <configuration>
-        <sourceFileIncludes>${project.basedir}/target/generated-sources/resgen/org/apache/calcite/runtime/Resources.java</sourceFileIncludes>
+        <additionalDependencies>
+          <additionalDependency>
+            <groupId>org.apache.calcite</groupId>
+            <artifactId>calcite-core</artifactId>
+            <version>1.24.0</version>
```

Comment:
Could you please add some more context on this version number?

Reply:
Good point, this is the recent version of org.apache.calcite. But it is a valid question for us on maintaining this dependency in future.